### DISCUSSION TITLE
Mobile detection

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#489] [Ideas]
Hi Rejetto,

How about mobile phone detection regarding the layout? Or can that be done in one of the HTML Lay-out areas?... I just changed the wallpaper plugin a bit to have a static background and a scrolling content, but now my mobile layout background is too short. :-)

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i'm not sure what you mean.
you need advice with css?
show me what you got

**Reply by PH5HP** [reply]
Hi Rejetto,

I was playing with the wallpaper plugin. I changed it so that the content of the server is scrolling over a static wallpaper. I discovered however that on my mobile phone the wallpaper did not strech like in my Google Chrome browser on my desktop computer.

But I think I found a solution which was much easier than I thought. Just changed no-repeat in repeat. See my screenshot attached. It looks a lot better now regarding the background wallpaper on my mobile device.

So solved for now (I hope), but I was wondering if there was a way to detect a mobile device and if detected changing the layout a bit if needed. I know that some people changinging layout regarding the resolution of the device on which it is displayed, but I learned tonight that doing that is not full-proof. So is there a solid way to detect a mobile device if needed?

Kind regards,
Henk - https://fileserver.ph5hp.nl
![image](https://github.com/rejetto/hfs/assets/140949730/2accbafa-e632-40f9-99fa-1c208a2b6ba3)

**Comment by rejetto** [comment]
with mobile i guess you mean a phone in portrait mode.
you should surround your rules in somthing like this.

@media (max-width: 30em) {
}

google and you'll find more info about it

**Reply by PH5HP** [reply]
Hi Rejetto,

Thank you ... I will get into this...  :-)

Kind regards,
Henk - https://fileserver.ph5hp.nl/
